scripts/integration.sh

```diff
@@ -0,0 +1,31 @@
+case $1 in
+  mac)
+    find ~/Library/Logs/Lens -type f -name *.log -delete
+  ;;
+  linux)
+    find ~/.config/Logs/Lens -type f -name *.log -delete
+  ;;
+  win)
+    find %APPDATA%/Logs/Lens -type f -name *.log -delete
+  ;;
+esac
+
+make build-extension-types build-extensions
+yarn build:$1
+DEBUG=true yarn integration-runner
```

Comment:
what is `integration-runner`?

Reply:
See `package.json`

Reply:
I don't see it. What am I looking for?

Reply:
Oh I see, I forgot to cherry pick that in, good catch.